src/main/java/org/mule/runtime/dsl/api/ConfigResource.java

```diff
@@ -38,10 +39,12 @@ public final class ConfigResource {
     String classPath = getProperty("java.class.path");
     String modulePath = getProperty("jdk.module.path");
 
+    String classPathSeparator = IS_OS_WINDOWS ? ";" : ":";
```

Comment:
https://stackoverflow.com/questions/60211687/how-to-get-java-classpath-separator